<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDistributionInfoComponent</class>
 <widget class="QFrame" name="CDistributionInfoComponent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>307</width>
    <height>90</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>90</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Distribution info</string>
  </property>
  <layout class="QGridLayout" name="gl_DistributionInfo">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="lbl_CurrentVersion">
     <property name="text">
      <string>This version:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QComboBox" name="cb_Platforms"/>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="lbl_LatestVersion">
     <property name="text">
      <string>Latest:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLineEdit" name="le_LatestVersion">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lbl_Channel">
     <property name="text">
      <string>Channel:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="le_CurrentVersion">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="cb_Channels"/>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="lbl_Platforms">
     <property name="text">
      <string>Platform:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="pb_CheckForUpdates">
     <property name="text">
      <string>check again</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lbl_NewVersionInfo">
     <property name="text">
      <string>Nothing new</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLabel" name="lbl_NewVersionUrl">
     <property name="text">
      <string>URL goes here</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
